app/models/katello/glue/candlepin/product.rb

```diff
@@ -7,12 +7,14 @@ def self.included(base)
       base.class_eval do
         lazy_accessor :productContent, :multiplier, :href, :attrs,
           :initializer => lambda { |_s| convert_from_cp_fields(Resources::Candlepin::Product.get(cp_id)[0]) }
-        # Entitlement Certificate for this product
-        lazy_accessor :certificate,
-          :initializer => lambda { |_s| Resources::Candlepin::Product.certificate(cp_id, self.organization.label) },
+        # Subscription for this product - used for certificate and key
+        lazy_accessor :subscription,
```

Comment:
the name 'subscription' is a tad confusing here, as its just storing the certificates.  I'd probably prefer something like 'certificate_data' or 'certificate_and_key'.  Something like that :)